docs/administration/deployment/upgrading-db/index.md

```diff
@@ -0,0 +1,110 @@
+# Upgrading database
+
+If you are the REANA cluster administrator upgrading an existing REANA
+cluster deployment from one REANA release series to another, such as
+from 0.7.* to 0.8.0, there might be changes in the REANA database schema
+between the two versions. In these cases, it is necessary to run a
+database upgrade script using [`alembic`](https://alembic.sqlalchemy.org/en/latest/).
+
+The `reana-db` command-line tool is provided to ease the database
+upgrade tasks. The tool is included in the `REANA-Server` component.
+
+Two different procedures will be explained. The first, covers a quick
+database upgrade after a successful cluster upgrade. The second, covers
+a more detailed upgrade procedure, in case you want to have a better
+control or you need to troubleshoot some unexpected issues during the
+upgrade.
+
+## Quick upgrade procedure
+
+After you upgrade REANA by running `helm upgrade` succcessfully, run the
+following command:
+
+```console
+$ kubectl exec -i -t deployment/reana-server -c rest-api -- reana-db alembic upgrade
+```
+
+The output will show the accomplished upgrades, for example:
+
+```console
+$ kubectl default exec -i -t deployment/reana-server -c rest-api -- reana-db alembic upgrade
+INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+INFO  [alembic.runtime.migration] Will assume transactional DDL.
+INFO  [alembic.runtime.migration] Running upgrade  -> c912d4f1e1cc, Quota tables.
+INFO  [alembic.runtime.migration] Running upgrade c912d4f1e1cc -> ad93dae04483, Interactive sessions.
+INFO  [alembic.runtime.migration] Running upgrade ad93dae04483 -> 4801b98f6408, Job started and finished times.
+INFO  [alembic.runtime.migration] Running upgrade 4801b98f6408 -> f84e17bd6b18, Workflow complexity.
+INFO  [alembic.runtime.migration] Running upgrade f84e17bd6b18 -> 6568d7cb6710, storing full workflow workspace.
+```
```

Comment:
I guess here we could add one more sentence explaining users that the database at this point was migrated successfully, that it's done and that they don't need to do anything else, WDYT?